app/src/test/java/com/mathroule/demoapp/HelperTest.java

```diff
@@ -0,0 +1,22 @@
+package com.mathroule.demoapp;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class HelperTest {
+
+    @Test
+    public void getRandomButtonLabel_1() {
+        assertTrue(Helper.getRandomButtonLabel().startsWith("Button #"));
+    }
+
+    @Test
+    @Ignore("Failing test due to randomness")
+    public void getRandomButtonLabel_2() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/15d3b9565f791bb5c2bdb22540e7aa2ee8420984/app/src/test/java/com/mathroule/demoapp/HelperTest.java#L18"></span>The JUnit 4 test method name ‘getRandomButtonLabel_2’ doesn’t match ‘[a-z][a-zA-Z0-9]*’</td>
    </tr>
  </tbody>
</table>